Library/Homebrew/utils.rb

```diff
@@ -304,10 +303,7 @@ def which_all(cmd, path = ENV["PATH"])
   end
 
   def which_editor
-    editor = ENV.values_at("HOMEBREW_EDITOR", "HOMEBREW_VISUAL")
-                .compact
-                .reject(&:empty?)
-                .first
+    editor = Homebrew::EnvConfig.editor
```

Comment:
Given users may have `HOMEBREW_VISUAL` in their in their bash profile or whatever, do we want to retain compatibility here?

Reply:
It wasn't a documented variable and I don't see what it's doing outside of `EDITOR`. That said, we do copy `HOMEBREW_VISUAL` from `VISUAL` so I think I should probably handle that case here.

Reply:
> we do copy `HOMEBREW_VISUAL` from `VISUAL`

That's perhaps where I see its usage then.